foosball/games/models.py

```diff
@@ -62,3 +65,94 @@ def __str__(self):
             self.score,
             ", ".join(map(str, self.players.all()))
         )
+
+
+class Player(models.Model):
```

Comment:
I don't think that is a good idea. This is more of a player games profile type of model. The main User model we have is for user accounts. These two are separate concepts. Also, what if someone wants to switch to their custom User model for their installation? That would need to implement all the player profile fields too.